src/alcotest.ml

```diff
@@ -463,16 +463,25 @@ let list_tests t () =
 
 let is_ascii s = String.for_all Char.Ascii.is_valid s
 
-let err_ascii s =
-  let err =
-    Printf.sprintf
-      "%S is not a valid test label (it should be an ASCII string), skipping." s
-  in
-  Fmt.(pf stderr) "%a %s\n%!" red "Error:" err
+let contains_illegal_char s =
+  let disallowed = ['/'; '\\'] in
```

Comment:
I think I'd prefer to have a whitelist, e.g. change `is_ascii` to actually check that the test name contains only "valid" characters (e.g. `a-z` + `A-Z`  + `0-9` + a few chars (maybe `.` and '-`' and `_`). (and not `..`)

Reply:
Actually I'm fine to disallow `.` as well. Can be fixed easily in most of my libs anyway :-)

Reply:
Agreed. This doesn't deal with the case-insensitive filesystem problem mentioned by @hannesm (which I haven't tested, but seems plausible), but we can worry about that separately.

Reply:
Will wait until #157 has been merged to make use of the `re` depencency.